### PR TITLE
ORPHAN fix(go-collapse): Fix slots and component initialisation to no…

### PR DESCRIPTION
**Pull request by MaaAx**
… longer require item as prop

### CODE REVIEW

File: src/components/GoAccordion/GoCollapse.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
         </div>
         <div class="go-header__title">
           <slot name="header">
-            <h5 v-html="item.title"></h5>
+            <h5 v-if="item" v-html="item.title"></h5>
```
Comment (renanbronchart) [inline src/components/GoAccordion/GoCollapse.vue]: v-if="item.title" ?

File: src/components/GoAccordion/GoCollapse.vue [hunk 2]
```diff
@@ -54,6 +54,7 @@
         <slot name="body">
           <div
             class="go-body__container"
+            v-if="item"
```
Comment (renanbronchart) [inline src/components/GoAccordion/GoCollapse.vue]: v-if="item.body" ?

File: src/components/GoAccordion/GoCollapse.vue (hunk 1 above)
Comment (MaaAx) [reply src/components/GoAccordion/GoCollapse.vue]: I want to test if "item" is null or undefined, more than if title is.

File: src/components/GoAccordion/GoCollapse.vue (hunk 2 above)
Comment (MaaAx) [reply src/components/GoAccordion/GoCollapse.vue]: same as above.

File: src/components/GoAccordion/GoCollapse.vue (hunk 1 above)
Comment (renanbronchart) [reply src/components/GoAccordion/GoCollapse.vue]: But if item is not null or undefined, I think the tag go on.

Comment (MaaAx) [reply src/components/GoAccordion/GoCollapse.vue]: It does, I just tested it => this v-if is useless, i'll remove it.

**Pull request merged**